As 1398 documentation for get command (#182)
askui/askui: docs/docs/general/07-Troubleshooting/deprecated-endpoints.md

# Deprecated Endpoints

If you are using the askui library you may see this warning:

> `This endpoint will be deprecated. Please upgrade your askui version.`

Reviewer: which warnning you want to use the one already in the inference or this?

Author: I will use the one which is already available

> This means that you are using an old askui version and old endpoints of our API.
> We will not maintain old endpoints and deprecate them in the near future. 
> 
> You can update askui to use our new endpoints with following command:

Reviewer: ```suggestion
This means that you are using an askui version, which will stop working in the near future. 
You can upgrade your askui version with the  following command:
```

# Deprecated Endpoints

The askui library is sending requests with screenshots and/or instructions
to endpoints of our inference API to get annotations and commands.
We are starting to use our new `inference` endpoint for this kind of requests.
Earlier versions (`v0.3.2` and prior) are using the `predict-comand` and `annotate` endpoint
of the API.
The old endpoints are still working but we are not planning to maintain them.
Both endpoints will be deprecated in the future.

> Please upgrade your askui version to use our new endpoint if you are using askui `v0.3.2` and earlier:

Reviewer: I don't like that you explained too much to the user about our current problem. and you made this chapter too specific for the current situation. 
from my perspective, this should be generic ( it only explains that if you get this warning, you may need to upgrade in the near future with this command. )  
would like to have the opinion from the others @JohannesDienst-askui , @adi-wan-askui , @programminx-askui

Author: good point. going for something shorter which is more generic. Also less changes in the future with this way

Reviewer 2: A generic explanation is fine in my opinion

```shell

> npm update askui

Reviewer 2: Was the problem solved that caused trouble sometimes when updating like this?

Reviewer: good catch, 
yes this might not work. 
you can change it to  `npm install askui@latest`

Author: had also probles with update but Samirs suggestion works fine :+1:
